DESCRIPTION

```diff
@@ -1,8 +1,8 @@
 Package: RcppCCTZ
 Type: Package
 Title: 'Rcpp' Bindings for the 'CCTZ' Library
-Version: 0.2.6
-Date: 2019-08-03
+Version: 0.2.7
```

Comment:
I tend not to make releases from PRs quite yet.  Let's make tthat 0.2.6.1, and adjust the ChangeLog to one of those 'Roll minor version' entries as well?

Reply:
Absolutely, that makes sense.

Reply:
Btw, the reason a version change is good is that `nanotime` will depend on it in order for `period` to work correctly, so it's useful that we can request that specific version. So 0.2.6.1 works perfectly :)

Reply:
I have been using that three-digits-for-releases and four for interim versions _that are still differentiable_ for a really long time.  It really works. :)   And also with the `package_version` S3 class (and/or the `dpkg` package comparison).